test/OpenTelemetry.Instrumentation.AspNetCore.Tests/MetricTests.cs

```diff
@@ -64,9 +64,10 @@ void ProcessExport(Batch<MetricItem> batch)
                 }
             }
 
+            var processor = new PullMetricProcessor(metricExporter, true);
```

Comment:
Good call 😄. I wrote these tests first and did not originally think about how the PullMetricProcessor makes things easier.

Reply:
I also put TODOs saying provider.Flush() can be leveraged, but then Pull to our rescue!